compiler/luci-interpreter/src/kernels/Pack.cpp

```diff
@@ -76,9 +76,8 @@ void Pack::configure()
     }
   }
 
-  if (t0->element_type() == DataType::S32 || t0->element_type() == DataType::U8 ||
-      t0->element_type() == DataType::S8 || t0->element_type() == DataType::S16 ||
-      t0->element_type() == DataType::S64)
+  if (t0->element_type() == DataType::U8 || t0->element_type() == DataType::S8 ||
+      t0->element_type() == DataType::S16)
```

Comment:
```c++
t0->zero_point()
```
fails due to `DataType::S32` and `DataType::S64` tensors don't have quantize params